<launch>
    <arg name="odom_frame_id"  default="odom_combined"/>

    <node pkg="turn_on_robot_htc" type="htc_wheeltec_robot_node" name="htc_wheeltec_robot" output="screen" respawn="false">
        <param name="usart_port_name"    type="string" value="/dev/wheeltec_controller"/>  
        <param name="serial_baud_rate"   type="int"    value="115200"/>
        <param name="odom_frame_id"      type="string" value="$(arg odom_frame_id)"/> 
        <param name="robot_frame_id"     type="string" value="base_footprint"/> 
        <param name="gyro_frame_id"      type="string" value="gyro_link"/> 
    </node>

</launch>